install/playbooks/roles/certificates/tasks/main.yml

```diff
@@ -130,24 +132,25 @@
   when: is_valid.stdout == "0"
   include_tasks: gencert.yml
 
-- name: Restrict permissions for the root of LetsEncrypt certificate directory
+# These permissions are needed to avoid problems with the
+
+- name: Set the permissions for certificate directories
   file:
-    path: '{{ path.location }}'
+    path: '{{ path }}'
     state: directory
-    mode: '{{ path.mode | default("0755") }}'
+    mode: '0755'
```

Comment:
Is there a specific reason for `0755` instead of `0750`?

Reply:
No specific reason, except I wanted to be sure it worked. I will test with 750 soon.